homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -95,16 +110,59 @@ def _async_process_data(self):
 
         self._is_on = data[self._binary_sensor]
 
-        self._state = STATE_OFF
+        self._state = STATE_ON if self._is_on else STATE_OFF
         if self._binary_sensor == "dhw_state":
-            self._icon = FLOW_OFF_ICON
+            self._icon = FLOW_ON_ICON if self._is_on else FLOW_OFF_ICON
         if self._binary_sensor == "slave_boiler_state":
-            self._icon = IDLE_ICON
-        if self._is_on:
+            self._icon = FLAME_ICON if self._is_on else IDLE_ICON
+
+        self.async_write_ha_state()
+
+
+class PwNotifySensor(PwBinarySensor, BinarySensorEntity):
+    """Representation of a Plugwise Notification binary_sensor."""
+
+    def __init__(self, hass, api, coordinator, name, dev_id, binary_sensor, model):
+        """Set up the Plugwise API."""
+        super().__init__(api, coordinator, name, dev_id, binary_sensor, model)
+
+        self._binary_sensor = binary_sensor
+        self._hass = hass
+
+        self._is_on = False
+        self._icon = None
+        self._attributes = {}
+
+        self._unique_id = f"{dev_id}-notification"
```

Comment:
Please keep this the same as in plugwise-beta: `self._unique_id = f"{dev_id}-{binary_sensor}"`

Reply:
Adjusted accordingly

Reply:
Thanks!

Reply:
Well spotted 👍